.github/workflows/plugin-release.yml

```
@@ -0,0 +1,200 @@
+        echo "Modified plugins: $pluginList"
+        echo "Legality resources changed: $legalityChanged"
+      shell: pwsh
+    
+    - name: Create or update release
+      if: steps.get_changes.outputs.PLUGINS != '' || steps.get_changes.outputs.LEGALITY_CHANGED == 'True'
+      run: |
+        $version = "${{ steps.get_version.outputs.VERSION }}"
+        $releaseExists = "${{ steps.check_release.outputs.RELEASE_EXISTS }}"
+        $pluginList = "${{ steps.get_changes.outputs.PLUGINS }}"
+        $legalityChanged = "${{ steps.get_changes.outputs.LEGALITY_CHANGED }}"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)